### PR TITLE
BZ2079224 - Adding  to required SCC api parameters

### PR DESCRIPTION
**Pull request by kelbrown20**
For Versions: 4.6+
[Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=2079224)
Ready for QE: 
Preview: [API reference -> Security APIs -> SecurityContextConstraints [security.openshift.io/v1]](https://deploy-preview-45182--osdocs.netlify.app/openshift-enterprise/latest/rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.html)

### CODE REVIEW

**Review by maiqueb — APPROVED** [review]
/lgtm

Thank you.

**Comment by Amrita42** [comment]
lgtm

File: rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.adoc [hunk 1]
```diff
@@ -27,6 +27,7 @@ Required::
   - `allowHostPorts`
   - `allowPrivilegedContainer`
   - `readOnlyRootFilesystem`
+  - `volumes`
```
Comment (wangke19) [inline rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.adoc:30]: I extracted one 0000_03_security-openshift_01_scc.crd.yaml from 4.6 nightly payload, checked the results for required list as below, 
```
required:
      - allowHostDirVolumePlugin
      - allowHostIPC
      - allowHostNetwork
      - allowHostPID
      - allowHostPorts
      - allowPrivilegedContainer
      - allowedCapabilities
      - defaultAddCapabilities
      - priority
      - readOnlyRootFilesystem
      - requiredDropCapabilities
      - volumes
```

Comment (wangke19) [reply rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.adoc:30]: @kelbrown20 Please take a look above comments.

**Comment by kelbrown20** [comment]
Left comment in Bugzilla regarding the status of this bug. Leaving open for now

**Comment by kelbrown20** [comment]
Setting too close becuase we can not directly update API docs. More updates in Bugzilla linked in description 
Repo with list required list: https://github.com/openshift/api/blob/a6156965faae5ce117e3cd3735981a3fc0e27e27/security/v1/0000_03_security-openshift_01_scc.crd.yaml#L61

**Pull request closed without merging**